components/supervisor/cmd/tunnel.go

```diff
@@ -56,7 +57,7 @@ var closeTunnelCmd = &cobra.Command{
 	Short: "close the tunnel",
 	Args:  cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		localPort, err := strconv.Atoi(args[0])
+		localPort, err := num.ParseUint32(args[0])
```

Comment:
Why not https://golang.org/pkg/strconv/#ParseUint?

Reply:
fixed